source/extensions/quic_listeners/quiche/platform/quic_mem_slice_storage_impl.cc

```diff
@@ -0,0 +1,39 @@
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include "extensions/quic_listeners/quiche/platform/quic_mem_slice_storage_impl.h"
+
+#include "quiche/quic/core/quic_utils.h"
+
+namespace quic {
+
+QuicMemSliceStorageImpl::QuicMemSliceStorageImpl(const struct iovec* iov, int iov_count,
```

Comment:
what's the portability story here? There's a concurrent effort to port to Windows. Does iovec exist there? Should this use an os portability layer?

Reply:
I don't think Windows is in CI but it's worth circling back with people working on that to see if this would regress their efforts.

So this is an awkward situation of where I'm not sure if we should submit this and add more barriers to getting Windiows in the first place.

Reply:
Per Slack discussion with @mattklein123 you can break windows for now. Please leave a TODO indicating this may need a refactor for Windows.